extensions/amp-ad-network-fake-impl/0.1/amp-ad-network-fake-impl.js

```diff
@@ -59,15 +59,24 @@ export class AmpAdNetworkFakeImpl extends AmpA4A {
           // and the signature is "FAKESIG". This mode is only allowed in
           // `localDev` and primarily used for A4A Envelope for testing.
           // See DEVELOPING.md for more info.
-          const creative = this.transformCreativeLocalDev_(deserialized);
+          let creative = this.transformCreativeLocalDev_(deserialized);
```

Comment:
now i re-read the code. can we move the `transformCreativeLocalDev_` logic to server side? so that we have a correct AMP ad format over the wire. ideally, this should be part of `app.use('/a4a(|-3p)/'` middleware.

After that, we don't need this try-catch anymore, since we know the exact format over the wire.

again, more logic in this fake extension, the farther away we are from testing the real things.